src/Microsoft.Identity.Web/Resource/AadIssuerValidator.cs

```diff
@@ -191,6 +191,12 @@ private static string GetTenantIdFromToken(SecurityToken securityToken)
                     return tid;
                 }
 
+                jsonWebToken.TryGetPayloadValue(ClaimConstants.TenantId, out string? tenantId);
```

Comment:
Does this also have to be added inside the above `if (securityToken is JwtSecurityToken jwtSecurityToken)`?

Reply:
yes. good catch.

Reply:
Are you looking to provide support for JsonWebToken as well? I see that on some places you have support for both JwtSecurityToken and JsonWebToken.